hbase-client/src/main/java/org/apache/hadoop/hbase/client/MetricsConnection.java

```diff
@@ -452,6 +455,10 @@ public void incrDelayRunnersAndUpdateDelayInterval(long interval) {
     this.runnerStats.updateDelayInterval(interval);
   }
 
+  public void incrementBackoffTime(long time, TimeUnit timeUnit){
```

Comment:
I feel like i should standardize on Ms here or convert all input times to Ms

Reply:
I can't see the full context of this atm (on mobile) but we should try to limit the scope of our changes. If we wqanted to change the units of all timers I think we'd want to do that in a separate issue

Reply:
Sorry, I wasn't super clear. I need to look into the timer code or if you know... cool.

So this is only for my backup timer... but in some situations when we increment this timer the timeUnit would be nanosec vs ms... so I wanted to check If the timer will automatically convert them all to ms so i can standardize the name to `backoffTimeMs` instead of just `backoffTime`. I also wanted to guarantee the unit so when we pull the metric in from the metrics collector we know what timeunit to expect.

Reply:
Confirmed everything will be converted to nanosecs. 

```java
  public void update(long duration, TimeUnit unit) {
        update(unit.toNanos(duration));
    }
.
.
.

 private void update(long duration) {
        if (duration >= 0) {
            histogram.update(duration);
            meter.mark();
        }
    }
```